src/lsstseries/ensemble.py

```diff
@@ -575,6 +633,22 @@ def from_parquet(
             Identifies which column contains the flux/mag error information
         band_col: 'str', optional
             Identifies which column contains the band information
+        nobs_col: list of 'str', optional
+            Identifies which columns contain number of observations for each
+            band, if available in the input object file
+        nobs_tot_col: 'str', optional
+            Identifies which column contains the total number of observations,
+            if available in the input object file
+        provenance_col: 'str', optional
+            Identifies which column contains the provenance information, if
+            None the provenance column is generated.
+        provenance_label: 'str', optional
```

Comment:
As per your other design document, the list of column names is getting unwieldy. Definitely not worth changing as part of this PR, but it would simplify this code a lot in the future.

Reply:
Agreed.